store/store.go

```diff
@@ -2196,7 +2196,11 @@ func genInstanceKey(curSnap *CurrentSnap, salt string) (string, error) {
 	return fmt.Sprintf("%s:%s", curSnap.SnapID, enc), nil
 }
 
-func (s *Store) snapAction(ctx context.Context, currentSnaps []*CurrentSnap, actions []*SnapAction, user *auth.UserState, opts *RefreshOptions) ([]*snap.Info, error) {
+type SnapActionResult struct {
```

Comment:
I think this type warrants a comment. Just getting the justification for its existence here would be enough.

Reply:
i've added it to the follow-up